docs/en/rules/Azure.APIM.Ciphers.md

```diff
@@ -21,6 +21,88 @@ These Ciphers are enabled by default and need to be set to False.
 ## RECOMMENDATION
 
 Consider disabling weak or deprecated ciphers.
+## EXAMPLES
+
+### Configure with Azure template
+
+
+
+For example:
+
+```json
+
+  {
+  "apiVersion": "2021-01-01-preview",
+  "name": "apim-contoso-test-001",
+  "type": "Microsoft.ApiManagement/service",
+  "location": "[resourceGroup().location]",
+  "tags": {},
+  "sku": {
+    "name": "Standard",
+    "capacity": "1"
+  },
+  "properties": {
+    "publisherEmail": "noreply@contoso.com",
+    "publisherName": "Contoso",
+    "customProperties": {
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TripleDes168": "False", # <----------------- ciphers TripleDes168 disabled  
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_RSA_WITH_AES_128_CBC_SHA": "False", # <----------------- ciphers TLS_RSA_WITH_AES_128_CBC_SHA disabled  
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_RSA_WITH_AES_256_CBC_SHA": "False", # <----------------- ciphers TLS_RSA_WITH_AES_256_CBC_SHA disabled  
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_RSA_WITH_AES_128_CBC_SHA256": "False", # <----------------- ciphers TLS_RSA_WITH_AES_128_CBC_SHA256 disabled
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA": "False", # <----------------- ciphers TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA disabled
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_RSA_WITH_AES_256_CBC_SHA256": "False", # <----------------- ciphers TLS_RSA_WITH_AES_256_CBC_SHA256 disabled
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA": "False", # <----------------- ciphers TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA disabled
+        "Microsoft.WindowsAzure.ApiManagement.Gateway.Security.Ciphers.TLS_RSA_WITH_AES_128_GCM_SHA256": "False" # <----------------- ciphers TLS_RSA_WITH_AES_128_GCM_SHA256 disabled
```

Comment:
Nice. Another suggestion would be to pull these out as separate dot points above the snippet, otherwise due to scrolling it might be hard for people to see this info.

For example:

![image](https://user-images.githubusercontent.com/13513058/191418289-74e8275a-1bad-4d75-84b2-50aafa8bbd19.png)

Reply:
updated the comments to move to the above the snippet.